doxygen warning

**Describe the bug**
psp/fsw/inc/cfe_psp_configdata.h:22: warning: the name `cfe_psp_config.h' supplied as the second argument in the \file statement is not an input file

**To Reproduce**
Steps to reproduce the behavior:
1. make usersguide
2. See error

**System observed on:**
 - Hardware
 - Ubuntu 18.04
 - doxygen 1.8.13, rc-6.7.0

**Reporter Info**
Anh Van, NASA Goddard

--- a/fsw/inc/cfe_psp_configdata.h
+++ b/fsw/inc/cfe_psp_configdata.h
@@ -19,7 +19,7 @@
 */
 
 /**
- * \file cfe_psp_config.h
+ * \file cfe_psp_configdata.h
  *
  *  Created on: Dec 31, 2014
  *      Author: joseph.p.hickey@nasa.gov

